source/_integrations/ozw.markdown

```diff
@@ -65,3 +65,24 @@ this operation.
 | Service Data Attribute | Required | Description                                        |
 | ---------------------- | -------- | -------------------------------------------------- |
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
```

Comment:
"code_slot" can be written as normal words, or, if really needed to be a reference to the variable, it needs to be code fenced e.g.,: ``` `code_slot` ```